.github/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,35 @@
+As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery
+* Personal attacks
+* Trolling or insulting/derogatory comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic addresses, without explicit permission
+* Other unethical or unprofessional conduct
+
+Project members with the Committer role have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+By adopting this Code of Conduct, project maintainers commit themselves to fairly and consistently applying these principles to every aspect of managing this project. Project maintainers who do not follow or enforce the Code of Conduct may be permanently removed from the project team.
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
+
+
+## Reporting
+
+Project members with the Committer role or the CRAN Maintainer role are pledged to promptly address any reported issues. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported to any individual with this role.
+
+Those who prefer to report in a way that is anonymous to the project maintainers may instead contact the Community Engagement Coordinator by e-mailing [r.data.table\@gmail.com](mailto:r.data.table@gmail.com). Messages sent to this e-mail address will be visible only to the current Community Engagement Coordinator, a position always held by an individual who is not a Committer or CRAN Maintainer of the package.
+
+The current Committers are Toby Dylan Hocking (@tdhock), Matt Dowle (@mattdowle), Arun Srinivasan (@arunsrinivasan), Jan Gorecki (@jangorecki), Michael Chirico (@MichaelChirico), and Benjamin Schwendinger (@ben-schwen).
```

Comment:
This is out of date (Ivan K. is recently added); rather than keep a manual list, should we point to this link instead?

https://github.com/orgs/Rdatatable/teams/committers

It's less important for the CRAN Maintainer & CEC roles because they're only one person, but still Tyson can also be found as "Release manager":

https://github.com/orgs/Rdatatable/teams/release-managers

And we could add a team for Kelly as Community Engagement Coordinator

Reply:
These links would be ideal but they are only visible to org members. :(

Reply:
Oh, I see. Hmm. Well, let's try both then:

> The current Committers (@Rdatatable/committers) are ...

That points to the group `@Rdatatable/committers` as a source-of-truth even if this manual list gets out of date.

Reply:
I suggest using DESCRIPTION which is public and should be up to date with github teams.

Reply:
DESCRIPTION has all Members though, where this is about Committers

Reply:
right, role="aut" in DESCRIPTION  <=> Committer Team on github.
the only issue we discussed about using DESCRIPTION is that committers would need to add their emails. I will submit a PR.